src/LSST/control/PowerSystem.cpp

```diff
@@ -59,16 +72,31 @@ void PowerSystem::_daqInfoRead() {
     // Get the latest info, old info is of no use.
     auto sInfo = _fpgaIo->getSysInfo();
     auto diff = (util::timePassedSec(sInfo.timestamp, _daqReadTime));
-    _checkTimeout(diff); // shutoff power if timed out.
+    bool timedOut = _checkTimeout(diff); // shutoff power if timed out.
+    if (timedOut) {
+        stringstream os;
+        time_t tm = util::CLOCK::to_time_t(_daqReadTime);
+        os << "PowerSystem::_daqInfoRead() timedOut last _daq read=" << ctime(&tm)
+                << " seconds since last read=" << diff;
+        LERROR(os.str());
+    }
 
     _processDaq(sInfo);
 }
 
 void PowerSystem::_daqTimeoutCheck() {
-    auto diff = (util::timePassedSec( _daqReadTime, util::CLOCK::now()));
+    auto now = util::CLOCK::now();
+    auto diff = (util::timePassedSec(_daqReadTime, now));
     bool timedOut = _checkTimeout(diff); // shutoff power if timed out.
     if (timedOut) {
+        stringstream os;
+        time_t tm = util::CLOCK::to_time_t(_daqReadTime);
+        os << "PowerSystem::Timeout timedOut last _daq read=" << ctime(&tm)
+                << " seconds since last read=" << diff;
```

Comment:
Is the format correct for the `<< " seconds since last read=" << diff;`? See the "space" compared with the line 94.